### ISSUE TITLE
Build problems on OS X 10.11

### ISSUE THREAD

**Issue by hannosch** [collective/buildout.python#49]
I've added the `darwin-elcapitan` platform and things mostly work, though I'm not building Python 2.4 nor 2.5 anymore.

The one problem I ran into was openssl. Python compilation couldn't find openssl in the system. It seems Homebrew had the same problem and [explicitly links to the homebrew provided openssl](https://github.com/Homebrew/homebrew/commit/80c83ab43a8f1946c301291985c8f2b1fbf073f9) instead.

I did the same locally by expanding the environment for each Python version:
```
[python-2.6-build:default]
environment =
    LDFLAGS=-L/usr/local/opt/openssl/lib
    CPPFLAGS=-I/usr/local/opt/openssl/include
```
But didn't want to commit that into the default buildout, as it depends on an outside non-standard library.

There's also another issue with "stub libraries" mentioned at https://bugs.python.org/issue25136, [with a patch](https://bugs.python.org/file40479/xcode-stubs-2.7.patch). This only seems to be required if the command line tools aren't installed explicitly via ``xcode-select --install``.

**Comment by mauritsvanrees** [comment]
@jladage This may give you some hints.

**Comment by jladage** [comment]
Hi @mauritsvanrees this helps for python2.6 and up, indeed. But python2.4 has problems with clang options. Maybe we should just drop python2.4 on El Capitan, old sites need to upgrade anyways since Plone 3 support is now drop because of the Plone 5 release.

**Comment by jladage** [comment]
With a git pull and brew link --force openssl I now have all pythons working.

**Comment by aclark4life** [comment]
Drop Python 2.4 :+1:

**Comment by mauritsvanrees** [comment]
Fixed by Hanno in f356afa96fb7fbda26af93eba0e562b95aeb26da.

**Comment by aclark4life** [comment]
Drop Python 2.4 :-1: (Changing my mind.) Why? Because I needed Python 2.4 just now on El Capitan and it's too hard and annoying to install without ``buildout.python``. I ran `` brew link --force openssl `` as @jladage suggested.
